packages/web-components/fast-components/src/listbox-option/listbox-option.styles.ts

```diff
@@ -62,7 +62,6 @@ export const OptionStyles = css`
     }
 
     :host(:not([aria-selected="true"]):hover) {
```

Comment:
Will this change cause the hover effect to go away for enabled options that are not selected? Should this instead be based on the disabled attribute?

Reply:
Hmm I rushed this. It was working on my end, but you are correct, updating now.